### PR TITLE
Fix bug in serialisation of AttributeView

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by chenkasirer — APPROVED** [review]
LGTM!

File: tests/compas/data/test_json.py [hunk 1]
```diff
@@ -121,12 +121,34 @@ def test_json_zip():
...
+
+    assert before.__dtype__ == after.__dtype__
+    assert all(before.has_vertex(vertex) for vertex in after.vertices())
+    assert all(after.has_vertex(vertex) for vertex in before.vertices())
+    assert all(before.has_face(face) for face in after.faces())
+    assert all(after.has_face(face) for face in before.faces())
+    assert all(before.has_edge(edge) for edge in after.edges())
+    assert all(after.has_edge(edge) for edge in before.edges())
+    assert all(before.face_vertices(a) == after.face_vertices(b) for a, b in zip(before.faces(), after.faces()))
+    assert before.guid == after.guid
```
Comment (chenkasirer) [inline tests/compas/data/test_json.py:149]: where are `AttributeView`s added here to the mesh?

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fixed bug in `Box.scaled` causing a `TypeError` due to incorrect parameter forwarding.
 * Changed argument names of `Box.scale()` to `x`, `y`, `z`, instead of `factor` and made `y` and `z` optional to keep positional arguments backwards compatible.
 * Fixed import errors in `compas_rhino.conduits` for Rhino 8.
+* Fixed bug in serialisation when `compas.datastructures.attributes.AttributeView` is used.
```
Comment (chenkasirer) [inline CHANGELOG.md]: sorry ;)
```suggestion
* Fixed bug in serialization when `compas.datastructures.attributes.AttributeView` is used.
```

File: tests/compas/data/test_json.py (hunk 1 above)
Comment (tomvanmele) [reply tests/compas/data/test_json.py:149]: whenever you ask for attributes of en element but don't specify which ones, you automatically get a view of the entire dict

**Pull request merged**